wp-includes/widgets/class-wp-widget-image.php

```diff
@@ -213,6 +215,41 @@ public function render_media( $instance ) {
 	}
 
 	/**
+	 * Filters the default media display states for items in the Media list table.
+	 *
+	 * @since 4.8.0
+	 * @access public
+	 *
+	 * @param array   $states An array of media states.
+	 * @param WP_Post $post   The current attachment object.
+	 * @return array
+	 */
+	public function display_media_state( $states, $post ) {
```

Comment:
> use $this->name instead of 'Image Widget'

Duh! I didn't think of that. Yes it should be in `WP_Widget_Media`, the only reason it's not is because I didn't want a hard coded array of names. So yeah, that solves it.